containers/roles/foreman/files/wait_on_postgres.py

```diff
@@ -10,7 +10,7 @@
     s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
     tries = 0
     print ("Waiting on postgresql to start...")
-    while not postgres_is_alive and tries < 20:
+    while not postgres_is_alive and tries < 100:
```

Comment:
Why is this a different value than the one in candlepin?

Reply:
Hah, fair question. I think I just kept adjusting them till things worked consistently.